src/intro.md

```diff
@@ -189,27 +187,33 @@ And now we are ready to install. Let's update our packages:
 sudo apt-get update
 ```
 
-Finally this installs a basic set of packages that should get you going with RMF:
+Finally, the following installs a basic set of packages that should get you going with RMF:
 
 ```
 sudo apt-get install ros-eloquent-ament-cmake-catch2 ros-eloquent-building-gazebo-plugins ros-eloquent-building-map-msgs ros-eloquent-building-map-tools ros-eloquent-rmf-cmake-uncrustify ros-eloquent-rmf-dispenser-msgs ros-eloquent-rmf-door-msgs ros-eloquent-rmf-fleet-adapter ros-eloquent-rmf-fleet-msgs ros-eloquent-rmf-lift-msgs ros-eloquent-rmf-task-msgs ros-eloquent-rmf-traffic-msgs ros-eloquent-rmf-traffic-ros2 ros-eloquent-rmf-traffic ros-eloquent-rmf-utils ros-eloquent-traffic-editor
 ```
 
-### Install and run RMF demos
+## Install and run RMF demos
 
-You can install the provided RMF demos from their debian package:
+You can install the provided RMF demos from their Debian package:
 
 ```
 sudo apt-get install ros-eloquent-demos
 ```
 
-In order to run ROS 2 commands we source the setup.bash file:
+If ROS 2 is not installed using at least `ros-eloquent-ros-base` you will also need the following packages:
```

Comment:
I think this would work better at the end of the "Setup ROS Sources" section. Because you have told readers who already have ROS installed to skip that section, it makes sense to assume those who need to set up their sources also need to install the necessary ROS packages.

Reply:
Will move it, thanks.